src/apps/mpx.erl

```diff
@@ -32,23 +32,27 @@ init([]) ->
   ?D(<<"Starting multiplex application">>),
   {ok, #state{}}.
 
-authorize(Client,_) -> {ok, Client#de_client{data=#{}, encoder=mpx, meta=[]}}.
+authorize(Client, Data) ->
+  case deliverly_utils:auth_from_config(mpx, Client, Data) of
+    true -> {ok, Client#de_client{data = #{}, encoder = mpx, meta = []}};
+    false -> {error, 3401}
+  end.
 
 handle_message(Data, Context) -> gen_server:call(?SERVER, {handle_message, Data, Context}).
```

Comment:
Line 41 is too long: handle_message(Data, Context) -> gen_server:call(?SERVER, {handle_message, Data, Context})..